plans/session-recording.fmf

```diff
@@ -7,3 +7,13 @@ adjust:
     - how: install
       name: Enable the Copr-repo for Requre
       copr: packit/packit-requre-main
+prepare:
+  - name: Install Git
+    how: install
+    package: git
+  - name: Configure Git
+    how: shell
+    script: |
+      git config --global safe.directory '*'
+      git config --global user.email "test@example.com"
```

Comment:
won't this break hardcoded user in tests? if there is any, but I guess, it's fine, if it already passed the tests :D